src/main.js

```diff
@@ -74,16 +74,21 @@ const refreshTimer = () => {
 };
 
 const changeColorBar = () => {
-  for (var i = 0; i < savedActivities.length; i++) {
-    if (savedActivities[i].category === 'Study') {
-      savedActivities[i].color = 'green-bar';
-    } else if (savedActivities[i].category === 'Meditate') {
-      savedActivities[i].color = 'purple-bar';
-    } else if (savedActivities[i].category === 'Exercise') {
-      savedActivities[i].color = 'red-bar';
+  savedActivities.forEach(savedActivity => {
```

Comment:
Nice work with this switch statement, I liked you strategy of commenting it out and trying the switch statement to see if it was written correctly.